src/components/home/aboutUsBlurb.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import Image from "next/image";
+import aboutBlurb from "@/public/home/aboutusimg.webp";
+
+const AboutUsBlurb = () => {
+  return (
+    <div className="ml-[10vw] mt-[5vw] h-[40vw] w-[80vw] rounded-3xl border-4">
+      <div className="mt-[40vh] flex items-center justify-center">
+        <Image
+          src={aboutBlurb}
+          alt="Group Club Photo"
+          className="absolute mr-[35vw] w-[39vw]"
```

Comment:
```suggestion
          className="w-[39vw]"
```
the image does not need to be absolutely positioned